ipa-core/tests/compact_gate.rs

```diff
@@ -37,11 +37,13 @@ fn compact_gate_cap_8_no_window_semi_honest_plaintext_input() {
 }
 
 #[test]
+#[ignore]
 fn compact_gate_cap_8_no_window_malicious_encrypted_input() {
     test_compact_gate(IpaSecurityModel::Malicious, 8, 0, true);
 }
 
 #[test]
+#[ignore]
```

Comment:
same here - lets explain why they're turned off and when they will be turn back on again

Reply:
```suggestion
/// This test is turned off because of [`issue`]. 
///
/// This test will hang without `relaxed-dp` feature turned out until it is fixed
/// [`issue`]: https://github.com/private-attribution/ipa/issues/1298
#[ignore]
```